src/state-summary/state-card-climate.js

```diff
@@ -30,7 +30,7 @@ class StateCardClimate extends PolymerElement {
 
   static get stateInfoTemplate() {
     return html`
-    <state-info hass="[[hass]]" state-obj="[[stateObj]]" in-dialog="[[inDialog]]"></state-info>
+    <state-info hass="[[hass]]" state-obj="[[stateObj]]" in-dialog="[[inDialog]]" config="[[config]]"></state-info>
```

Comment:
Let's introduce new lines between attributes and not create such long lines. Also, we should not make these components config aware.